### PR TITLE
Update Huawei CSI Support

### PR DESCRIPTION
**Pull request by wuxueyi19**
Add some Huawei storage types.

### CODE REVIEW

File: book/src/drivers.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
...
 [HPE](https://github.com/hpe-storage/csi-driver) | `csi.hpe.com` | v0.3, v1.0, v1.1 | A Container Storage Interface (CSI) driver from HPE | Persistent and Ephemeral | Read/Write Single Pod | Yes | Raw Block, Snapshot, Expansion, Cloning
-[Huawei Storage CSI](https://github.com/Huawei/eSDK_K8S_Plugin) | `csi.huawei.com` | v1.0 | A Container Storage Interface (CSI) Driver for [Huawei Storage](https://support.huawei.com/enterprise/en/index.html) | Persistent | Read/Write Multiple Pod | Yes | 
+[Huawei Storage CSI](https://github.com/Huawei/eSDK_K8S_Plugin) | `csi.huawei.com` | v1.0 | A Container Storage Interface (CSI) Driver for [FusionStorage, OceanStor 100D, ](https://support.huawei.com/enterprise/en/category/cloud-storage-pid-1482615290338?submodel=21430817)[OceanStor Dorado V3, OceanStor Dorado V6, OceanStor V3, OceanStor V5](https://support.huawei.com/enterprise/en/category/enterprise-storage-pid-1482606575525?submodel=21430818) | Persistent | Read/Write Multiple Pod | Yes |
```
Comment (xing-yang) [inline book/src/drivers.md:44]: The display does not look right.  Can you take a look of "IBM Block Storage" as an example of showing multiple storage types?

Comment (xing-yang) [reply book/src/drivers.md:44]: See the display here.  It's messed up: https://github.com/kubernetes-csi/docs/blob/28bb27135a64d8b1d5d7acf8f995ca792f40bf88/book/src/drivers.md

**Comment by wuxueyi19** [comment]
We modified the docs and canceled the hyperlink. (There are 2 different hyperlinks for HuaweiStorage, but only one can be added.)

**Pull request merged**